src/unix/fs.c

```diff
@@ -736,6 +741,15 @@ static ssize_t uv__fs_sendfile_emul(uv_fs_t* req) {
   return nsent;
 }
 
+#ifdef __NR_copy_file_range
+static loff_t
+uv__fs_copy_file_range(int fd_in, loff_t *off_in, int fd_out,
+                loff_t *off_out, size_t len, unsigned int flags)
+{
+  return syscall(__NR_copy_file_range, fd_in, off_in,
+                 fd_out, off_out, len, flags);
+}
+#endif
```

Comment:
Guarding on `__NR_copy_file_range` is kind of debatable because we aim to make libuv forward compatible. When you compile libuv on an old system and copy it to a new system, it should use the new features that system supports.

Take a look at `src/unix/linux-syscalls.c` to see how we deal with that for other system call wrappers.